Snakefile

```diff
@@ -341,6 +341,29 @@ if config["enable"].get("retrieve_cost_data", True):
             move(input[0], output[0])
 
 
+rule build_demand:
```

Comment:
Maybe `build_demand_profile` to make in coherent with `build_renewable_profiles`

Reply:
Done!